elasticsearch/init/0200-seed-index-templates

```diff
@@ -17,6 +17,25 @@
 
 source "logging"
 
+info Deleting deprecated index templates
+shopt -s failglob
+for deprecated_file in ${ES_HOME}/index_templates/deprecated/*.json
+do
+    template=`basename $deprecated_file`
+    # Check if index template already exists
+    response_code=$(es_util --query=_template/$template \
+        ${DEBUG:+-v} -s \
+        --request HEAD --head --output /dev/null \
+        -w '%{response_code}')
+    if [ "${response_code}" == "200" ]; then
+        info "Index template '$template' found in the cluster, deleting it"
```

Comment:
Just a small nit, but maybe we can add the word "deprecated" into the log message.
> Deprecated index templates .... found ... blah, deleting it.
It makes is explicit if only a small snippet of the log output is shared ...